pkg/services/ngalert/models.go

```diff
@@ -0,0 +1,92 @@
+package ngalert
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/ngalert/eval"
+)
+
+// AlertDefinition is the model for alert definitions in Alerting NG.
+type AlertDefinition struct {
+	Id        int64
+	OrgId     int64
```

Comment:
Initialisms should have consistent case in Go.
```suggestion
	ID        int64
	OrgID     int64
```

Reply:
This is a model so I'm afraid I cannot change it because of the go-xorm naming convention.

I think I've tried also changing the `OrgId` -> `OrgID` but then it was looking for column `org_i_d`.

Reply:
> This is a model so I'm afraid I cannot change it because of the go-xorm naming convention.

@papagian Oh yeah, it's no biggie. I think there's a tag directive you can provide though, to instruct xorm wrt. the translated name.